src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -78,7 +75,12 @@ private Connector() {
         throw new IllegalAccessError("Utility class");
     }
 
-    public static ListBoxModel listScanCredentials(SCMSourceOwner context, String apiUri) {
+    /** Binary compatibility. */
+    public static ListBoxModel listScanCredentials(@CheckForNull SCMSourceOwner context, String apiUri) {
+        return listScanCredentials((Item) context, apiUri);
+    }
+
+    public static ListBoxModel listScanCredentials(@CheckForNull Item context, String apiUri) {
```

Comment:
Adding nullability annotations, though I could not get FindBugs to report errors anyway…not sure what kind of analysis it is doing.